cmd/client-s3-trace_v4.go

```diff
@@ -38,6 +38,15 @@ func newTraceV4() httptracer.HTTPTracer {
 func (t traceV4) Request(req *http.Request) (err error) {
 	origAuth := req.Header.Get("Authorization")
 
+	printTrace := func() error {
+		var reqTrace []byte
```

Comment:
this extra definition is not needed since you are using `reqTrace, rerr :=`

Reply:
done